sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateClient.cs

```diff
@@ -66,7 +66,10 @@ public CertificateClient(Uri vaultUri, TokenCredential credential, CertificateCl
             _pipeline = new KeyVaultPipeline(vaultUri, options.GetVersionString(), pipeline, new ClientDiagnostics(options));
         }
 
-        // Certificates API
+        /// <summary>
+        /// Gets the <see cref="Uri"/> of the vault used to create this instance of the <see cref="CertificateClient"/>.
+        /// </summary>
+        public Uri VaultUri => _pipeline.VaultUri;
```

Comment:
virtual for mocking.

Reply:
in all the clients.

Reply:
Okay. FWIW, our current instrumenting functionality for recording doesn't mock properties, but I can understand how third-party code might.